src/main/java/cinema/entity/Ticket.java

```diff
@@ -0,0 +1,16 @@
+package cinema.entity;
+
+import lombok.Value;
+
+import java.math.BigDecimal;
+
+@Value
```

Comment:
Why do you use https://github.com/value annotation here?

Reply:
replaced with @data annotation